src/client/services/avatar/avatar-url-batched-request-queue.js

```diff
@@ -1,62 +0,0 @@
-/**!
```

Comment:
We're not deleting anything yet; once the web client moves to the packages sdk, we'll get rid of src and test in one shot